### PR TITLE
Bugbash fixes

### PR DESCRIPTION
**Pull request by kanika1894**
1. Fixing failure due to space in input : https://github.com/github/azure-integration/issues/7
2. Removing unnecessary warning : https://github.com/github/azure-integration/issues/8

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -22,7 +22,6 @@ inputs:
   deploymentMode:
     description: "Incremental (only add resources to resource group) or Complete (remove extra resources from resource group)."
     required: false
-    default: Incremental
```
Comment (bishal-pdMSFT) [inline action.yml:25]: why this change?

File: src/deploy/scope_managementgroup.ts [hunk 2]
```diff
@@ -15,12 +15,12 @@ export async function DeployManagementGroupScope(azPath: string, validationOnly:
...
-        region ? `--location ${region}` : undefined,
+        region ? `--location "${region}"` : undefined,
         template ?
             template.startsWith("http") ? `--template-uri ${template}` : `--template-file ${template}`
             : undefined,
-        managementGroupId ? `--management-group-id ${managementGroupId}` : undefined,
-        deploymentName ? `--name ${deploymentName}` : undefined,
+        managementGroupId ? `--management-group-id "${managementGroupId}"` : undefined,
+        deploymentName ? `--name "${deploymentName}"` : undefined,
         parameters ? `--parameters ${parameters}` : undefined
```
Comment (bishal-pdMSFT) [inline src/deploy/scope_managementgroup.ts:24]: what if the value of a parameter needs to have spaces. Will user have to use double quotes and will that work?
For example `parameters: foo="bar baz"`?

File: action.yml (hunk 1 above)
Comment (bishal-pdMSFT) [reply action.yml:25]: ok, got it.

**Review by bishal-pdMSFT — APPROVED** [review]

File: src/deploy/scope_managementgroup.ts (hunk 2 above)
Comment (kanika1894) [reply src/deploy/scope_managementgroup.ts:24]: yes, that works
